help/Readme.md

```diff
@@ -0,0 +1,140 @@
+- `virtual_network_subnet_name` - If `virtual_network_name is set`, this value may also be set. If `virtual_network_name` is set, and this value is not set the builder attempts to determine the subnet to use with the virtual network. If the subnet cannot be found, or it cannot be disambiguated, this value should be set.
+- `capture_name_prefix` - VHD prefix. The final artifacts will be named PREFIX-osDisk.UUID and PREFIX-vmTemplate.UUID.
+- `github_feed_token` - Github PAT. Required for NPM toolcache installation. Will be depricated soon.
+
+#### Common vairables
+- `vm_size` - Size of the VM used for building. This can be changed when you deploy a VM from your VHD. See pricing information. https://docs.microsoft.com/en-us/azure/search/search-sku-tier
+- `image_os` - Type of OS that will be deployed as a temporary VM.
+- `image_version` -  Specify a specific version of an OS to boot from.
+- `imagedata_file` - Path to json file that will contain image-version and readme path
+- `helper_script_folder` - Path, where helper modules will be deployed.
+- `image_folder` - Root path of current image scripts location.
+
+#### Linux only
+- `metadata_file` - Path to metadata file that required for documentation creation.
+- `installer_script_folder` - Root path of installation scripts.
+- `run_validation_diskspace` - Boolean flag. Enable or disable available disk space scan.
+
+#### Windows only
+- `run_scan_antivirus` - Boolean flag. Enable or disable antivirus scan during image generation.
+- `root_folder` - Path to installation root folder.
+- `toolset_json_path` - Path to toolset config.
+- `psmodules_root_folder` - Path to AZ modules.
+- `install_user` - User under which all installations will be performed.
+- `install_password` - Installation user password.
+
+**Detailed Azure builders documentation can be found [here](https://www.packer.io/docs/builders/azure).**
+
+### Toolset
+To reduce image generation time, we store prebuilted versions of Python, Boost, Node and ruby in external sources, from which toolset-provision scripts download them during the build. If needed, unused versions or tools from toolset can be safely disabled, via removing from specified `toolset.json` config.
+
+Example of toolset configuration:
+```
+"name": "Python",
+"url" : "https://raw.githubusercontent.com/actions/python-versions/master/versions-manifest.json",
+"platform" : "linux",
+"platform_version": "20.04",
+"arch": "x64",
+"versions": [
+    "2.7.*",
+    "3.5.*",
+    "3.6.*",
+    "3.7.*",
+    "3.8.*"
+]
+```
+Generated tool versions and details can be found in related projects:
+- [Python](https://github.com/actions/python-versions/)
+- [Go](https://github.com/actions/go-versions)
+- [Node](https://github.com/actions/node-versions)
+- [Boost](https://github.com/actions/boost-versions)
```

Comment:
I think we should not include json example here because it is pretty huge and share details about Boost, Node, Python, Go. Because toolsets contain not only these tools but a lot of other tools, like powershell modules, VS components and etc.

I suggest to rephrase this section and do it very short:
```
Configuration for some installed software is located in `toolset.json` files. These files define the list of Ruby, Python, Go versions, the list of PowerShell modules and VS components that will be installed to image. They can be changed if these tools are not required to reduce image generation time or image size.
```

Reply:
Done